### PR TITLE
chore(content): remove classic theme - FE-3105

### PR DESCRIPTION
**Pull request by DipperTheDan**
removal of classic story from component, refactor stories file to CSF, create an mdx file and remove
redundant documentation

fixes FE-3105

### Proposed behaviour

- Remove redundant documentation
- Create an MDX example for Content
- Refactor stories file to be CSF
- Remove `classic` story

### Current behaviour

- Component still has our old style documentation
- No MDX example for Content
- Stories file is currently in `storiesOf` format
- `classic` story exists

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### Additional context
N/A

### Testing instructions
- `git checkout content_remove_classic_theme`
- `npm start` to start Storybook
- Navigate to Content
- Example should be in MDX and have no `classic` themed example

### CODE REVIEW

File: cypress/features/regression/content.feature [hunk 1]
```diff
@@ -21,7 +21,7 @@ Feature: Content component
 
   @positive
   Scenario Outline: Change property in Content dialog to <property>
-    When I open default "Content" component in noIFrame with "content" json from "commonComponents" using "<nameOfObject>" object name
+    When I open basic "Content Test" component in noIFrame with "content" json from "commonComponents" using "<nameOfObject>" object name
```
Comment (DlgSHi) [inline cypress/features/regression/content.feature:24]: we could delete this TC - it is covered in `chromatic`

File: cypress/features/regression/content.feature [hunk 2]
```diff
@@ -30,7 +30,7 @@ Feature: Content component
 
   @positive
   Scenario Outline: Change alignProperty in Content dialog to <alignProperty>
-    When I open default "Content" component in noIFrame with "content" json from "commonComponents" using "<nameOfObject>" object name
+    When I open basic "Content Test" component in noIFrame with "content" json from "commonComponents" using "<nameOfObject>" object name
```
Comment (DlgSHi) [inline cypress/features/regression/content.feature:33]: we could convert this TC - to be `chromatic`

File: cypress/features/regression/content.feature [hunk 3]
```diff
@@ -40,17 +40,17 @@ Feature: Content component
 
   @positive
   Scenario: BodyFullWidth enabled
-    When I open default "Content" component in noIFrame with "content" json from "commonComponents" using "bodyFullWidth" object name
+    When I open basic "Content Test" component in noIFrame with "content" json from "commonComponents" using "bodyFullWidth" object name
```
Comment (DlgSHi) [inline cypress/features/regression/content.feature:43]: we could convert this TC - to be `chromatic`

File: cypress/features/regression/content.feature [hunk 4]
```diff
@@ -61,5 +61,5 @@ Feature: Content component
 
   @positive
   Scenario: Inline disabled
-    When I open default "Content" component in noIFrame with "content" json from "commonComponents" using "inlineFalse" object name
+    When I open basic "Content Test" component in noIFrame with "content" json from "commonComponents" using "inlineFalse" object name
```
Comment (DlgSHi) [inline cypress/features/regression/content.feature:64]: we could delete this TC - it is covered by `chromatic`

File: src/components/content/content.stories.js [hunk 5]
```diff
@@ -1,67 +1,48 @@
...
+};
 
-  const metadata = {
-    themeSelector,
-    notes: { markdown: notes },
+basic.story = {
+  parameters: {
     chromatic: {
-      disable: disableChromatic,
+      disable: false,
```
Comment (DlgSHi) [inline src/components/content/content.stories.js:45]: it is also can be `true` - we have the same story rendered in `mdx`

File: src/components/content/content.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+## Quick Start
+
+```jsx
+import Content from "carbon-react/lib/components/content";
+```
+
+## Default Content
+<Preview>
+  <Story name="Default">
+    <Content title='Title' as='primary' children='This is an example of some content'/>
```
Comment (mkrds) [inline src/components/content/content.stories.mdx:21]:
```suggestion
    <Content title='Title' as='primary'>This is an example of some content</Content>
```
Do the same for other examples as well

**Comment by edleeks87** [comment]
Nothing else to add past what's already commented

**Review by mkrds — DISMISSED** [review]

**Review by DlgSHi — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Review by edleeks87 — DISMISSED** [review]

**Review by mkrds — DISMISSED** [review]

**Review by DlgSHi — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by mkrds — APPROVED** [review]

**Pull request merged**